system/settings/settings.xml

```diff
@@ -3022,14 +3022,14 @@
             <show more="true" details="true">installed</show>
           </control>
         </setting>
-        <setting id="lookandfeel.skinsettings" type="action" parent="lookandfeel.skin" label="21417" help="36104">
+        <setting id="lookandfeel.skinsettings" type="action" parent="lookandfeel.skin" label="14266" help="36104">
           <level>0</level>
           <dependencies>
             <dependency type="enable" on="property" name="AddonHasSettings" setting="lookandfeel.skin" />
           </dependencies>
           <control type="button" format="action" />
         </setting>
-        <setting id="lookandfeel.skintheme" type="string" parent="lookandfeel.skin" label="15111" help="36105">
+        <setting id="lookandfeel.skintheme" type="string" label="15111" help="36105">
```

Comment:
I was looking for a variation on option 1) to make "- Configure skin..." stand out better without resorting to using a separate group as in option 2) above.